lib/application/wallet/providers.dart

```diff
@@ -75,6 +75,7 @@ class _SessionNotifier extends Notifier<Session> {
   }
 
   Future<void> logout() async {
+    await ref.read(NotificationProviders.repository).unsubscribeAll();
```

Comment:
Globally, do you think NotificationProviders should be encapsulated by FeatureFlags Messenger  ?